eslint.config.js

```diff
@@ -1,15 +1,16 @@
-import globals from 'globals'
```

Comment:
your linter for some reason is adding semicolons and then adding dangling commas 🤔